<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataViewer</class>
 <widget class="QWidget" name="DataViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="tablesList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>256</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dbNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1071</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>69</y>
     <width>821</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QLabel" name="nameEditLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Назва бази даних:</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginEditLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Логiн:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passEditLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пароль:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="displayPassword">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показати пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbNameEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>hgfhfghf</string>
    </property>
    <property name="toolTipDuration">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbLoginEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dbPassEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="dbEditLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>-10</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Редагування бази даних</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dbEditSaveButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Зберегти змiни</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dbEditReloadButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Скасувати незбереженi 
 змiни</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="dbSaveButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти базу даних</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbSaveAsButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти базу даних як...</string>
   </property>
  </widget>
  <widget class="QWidget" name="tableWidget" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>831</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="dbTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>501</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="dbSaveTable">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зберегти таблицю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbExitButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Вихiд</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteTableButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Видалити
таблицю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dbCreateTable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>580</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cтворити таблицю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRowButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Видалити
вибранi
рядки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>210</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Додати
рядок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="makeProjection">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>280</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Проекцiя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IntersectionButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>280</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Перетин
таблиць</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
